ni_measurementlink_service/_internal/stubs/proto/ni/measurementlink/sessionmanagement/v1/session_management_service.proto

```diff
@@ -62,6 +62,17 @@ service SessionManagementService {
 
     // Unregisters the multiplexer sessions with this service. Indicates that the sessions have been closed and will need to be reopened before they can be used again.
     rpc UnregisterMultiplexerSessions(UnregisterMultiplexerSessionsRequest) returns (UnregisterMultiplexerSessionsResponse);
+
+    // Gets all the connected multiplexer session(s) for the given pin map context and returns information needed to create or access the session.
+    // Status Codes for errors:
+    // - INVALID_ARGUMENT:
+    //     - Pin Map Context references a site number that is not defined in the pin map.
+    // - NOT_FOUND:
+    //     - Pin Map Context has a pin map ID that does not match any pin maps registered with the Pin Map Service.
+    rpc GetMultiplexerSessions(GetMultiplexerSessionsRequest) returns (GetMultiplexerSessionsResponse);
+
+    // Gets all multiplexer sessions currently registered with this service.
```

Comment:
```suggestion
    // Gets all multiplexer sessions currently registered in the session management service.
```

Reply:
The term, "with this service" was preferred to maintain consistency with the other API (RegisterSessions, UnregisterSessions, ReserveAllRegisteredSessions) documentation in this proto.